.github/workflows/ci_test-slow.yml

```diff
@@ -0,0 +1,82 @@
+name: Test
+
+# see: https://help.github.com/en/actions/reference/events-that-trigger-workflows
+on:  # Trigger the workflow on push or pull request, but only for the master branch
+  push:
+    branches: [master, "release/*"]
+  pull_request:
+    branches: [master, "release/*"]
+
+jobs:
+  slow:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # same config as '.azure-pipelines/gpu-tests.yml'
+        python-version: ["3.7"]
+        pytorch-version: ["1.8"]
+
+    timeout-minutes: 20
```

Comment:
only 20min? sounds fast :]

Reply:
Because we don't have many "slow tests at the moment". I think this could be tuned lower

Reply:
Ubuntu took 16 min and mac 13. Seems like 20 is good for now